src/test/suite/fixtures/recorded/actions/roundWrapVest.yml

```diff
@@ -0,0 +1,30 @@
+spokenForm: round wrap vest
+languageId: typescript
+command:
+  actionName: wrap
+  partialTargets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: v}
+  extraArgs: [(, )]
+marks:
+  default.v:
+    start: {line: 1, character: 6}
+    end: {line: 1, character: 21}
+initialState:
+  documentContents: |
+
+    const value_hello_str = "Hello world";
+  selections:
+    - anchor: {line: 1, character: 21}
+      active: {line: 1, character: 21}
+finalState:
+  documentContents: |
+
+    const (value_hello_str) = "Hello world";
+  selections:
+    - anchor: {line: 1, character: 22}
```

Comment:
Shouldn't this be 23? Because two chars were inserted to the left of cursor?

Reply:
This is on purpose. I made special considerations for the right hand insert in the wrap command. If the right hand insertion is at the cursor don't move the cursor. This is we actually can wrap the cursor and have the curses stay inside the wrapt area afterwards.
`round wrap this` => `(|)`
`round wrap vest` => `(value|)`